playbooks/roles/csi-driver-install/files/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
This may be a template file when used to substitute the IBMCLOUD_API_KEY variable.